### PR TITLE
Add Qt-Designer

### PR DESCRIPTION
**Pull request by ivaquero**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by aaronliu0130** [comment]
Duplicate of [qt-creator](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/qt-creator.rb). fman.io also isn't qt's official site, it should be qt.io

**Comment by ivaquero** [comment]
> Duplicate of [qt-creator](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/qt-creator.rb). fman.io also isn't qt's official site, it should be qt.io

This one is an independent app. It's not an official product of Qt project.

**Comment by aaronliu0130** [comment]
No, it is.
https://doc.qt.io/qt-6/qtdesigner-manual.html
https://en.wikipedia.org/wiki/Qt_Creator?lang=en#:~:text=the%20integration%20of%20Qt%20Designer%20under%20Qt%20Creator%20is%20first%20mentioned%20at%20least%20as%20early%20as%20Qt%204.7%20(ca.%20late%202011).%5B7%5D%20In%20the%20Qt%205%20era%2C%20it%20is%20simply%20stated%20that%20%22%5BQt%20Designer%27s%5D%20functionality%20is%20now%20included%20as%20part%20of%20%5Bsic%5D%20Qt%20Creator%20IDE.

**Comment by ivaquero** [comment]
> https://doc.qt.io/qt-6/qtdesigner-manual.html

Thank you for explanation. I saw this app linked to a personal GitHub repo, and I guessed it was a personal build of the Designer. 

However, will this be considered and accepted as a lite-version of qt-creator ?

**Comment by aaronliu0130** [comment]
https://docs.brew.sh/Acceptable-Casks#unofficial-vendorless-and-walled-builds
> ### Unofficial, Vendorless, and Walled Builds
> We do not accept these casks since they offer a higher-than-normal security risk.

File: Casks/qt-designer.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "qt-designer" do
+  version :latest
+  sha256 :no_check
+
+  url "https://build-system.fman.io/static/public/files/Qt%20Designer.dmg"
+  name "Qt Designer"
+  desc "Create Python GUIs with Qt in minutes"
```
Comment (aaronliu0130) [inline Casks/qt-designer.rb:7]: 1. "in minutes" is marketing jargon that should be removed
2. Qt Designer isn't limited to python

**Comment by ivaquero** [comment]
> https://docs.brew.sh/Acceptable-Casks#unofficial-vendorless-and-walled-builds
> 
> > ### Unofficial, Vendorless, and Walled Builds
> 
> > We do not accept these casks since they offer a higher-than-normal security risk.

Thank you again, I will keep this for my personal use.

**Pull request closed without merging**